lang/csharp/src/apache/test/Schema/SchemaNormalizationTests.cs

```diff
@@ -50,15 +50,15 @@ public void FingerprintTest(string input, string expectedOutput)
 
         private static List<object[]> ProvideFingerprintTestCases()
         {
-            using (StreamReader reader = new StreamReader("../../../../../share/test/data/schema-tests.txt"))
+            using (StreamReader reader = new StreamReader("../../../../../../../../share/test/data/schema-tests.txt"))
```

Comment:
So many dots on the dance floor!

Reply:
TBH this could just be an artifact of how I'm running the tests, but the tests all need to be upgraded to work well with new tooling anyway, so I didn't spend much time thinking about it.